### PR TITLE
[docs] Refine SSHCredentials docs

### PR DESCRIPTION
**Pull request by asviel**
## Description

Refine docs.

## Why do we need it, and what problem does it solve?

It's not clear why `SSHCredentials` resource is needed.

## Why do we need it in the patch release (if we do)?

If we want these improvements to become available to users sooner.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: chore
summary: Docs improvements for CAPS.
impact_level: low
```

### CODE REVIEW

File: modules/040-node-manager/crds/doc-ru-sshcredentials.yaml [hunk 1]
```diff
@@ -4,9 +4,9 @@ spec:
       schema:
         openAPIV3Schema:
           description: |
-            Содержит данные для подключения по SSH.
+            Содержит данные SSH, необходимые для подключения к хосту для первоначальной настройки при использовании Cluster API Provider Static.
```
Comment (z9r5) [inline modules/040-node-manager/crds/doc-ru-sshcredentials.yaml:7]:
```suggestion
            Содержит данные, необходимые для подключения по SSH от Cluster API Provider Static к серверу (виртуальной машине), описанному в ресурсе [StaticInstance](cr.html#staticinstance).
```

File: modules/040-node-manager/crds/doc-ru-sshcredentials.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ spec:
...
 
-            Ссылка на этот ресурс указывается в ресурсе `StaticInstance`.
+            `StaticInstance` ссылается на этот ресурс.
```
Comment (z9r5) [inline modules/040-node-manager/crds/doc-ru-sshcredentials.yaml:9]:
```suggestion
             Ссылка на этот ресурс указывается в параметре [credentialsRef](cr.html#staticinstance-v1alpha1-spec-credentialsref) ресурса `StaticInstance`.
```

File: modules/040-node-manager/crds/sshcredentials.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ spec:
       schema:
         openAPIV3Schema:
           description: |
-            SSHCredentials is the Schema for the SSH credentials to connect to the hosts via SSH.
+            Contains the SSH credentials required to connect to the host for bootstrap when using Cluster API Provider Static.
```
Comment (z9r5) [inline modules/040-node-manager/crds/sshcredentials.yaml:24]:
```suggestion
                        Содержит данные, необходимые Cluster API Provider Static (CAPS) для подключения по протоколу SSH. CAPS подключается к серверу (виртуальной машине), описанному в ресурсе [StaticInstance](cr.html#staticinstance) для управления его состоянием.

                        Ссылка на этот ресурс указывается в параметре [credentialsRef](cr.html#staticinstance-v1alpha1-spec-credentialsref) ресурса `StaticInstance`.

```

File: modules/040-node-manager/crds/sshcredentials.yaml [hunk 4]
```diff
@@ -21,7 +21,7 @@ spec:
...
 
             The `StaticInstance` resource references this resource.
```
Comment (z9r5) [inline modules/040-node-manager/crds/sshcredentials.yaml:26]:
```suggestion
            Содержит данные, необходимые Cluster API Provider Static (CAPS) для подключения по протоколу SSH. CAPS подключается к серверу (виртуальной машине), описанному в ресурсе [StaticInstance](cr.html#staticinstance) для управления его состоянием.
            Ссылка на этот ресурс указывается в параметре [credentialsRef](cr.html#staticinstance-v1alpha1-spec-credentialsref) ресурса `StaticInstance`.
```

File: modules/040-node-manager/crds/sshcredentials.yaml [hunk 5]
```diff
@@ -21,9 +21,8 @@ spec:
       schema:
         openAPIV3Schema:
           description: |
-            SSHCredentials is the Schema for the SSH credentials to connect to the hosts via SSH.
-
-            The `StaticInstance` resource references this resource.
+            Содержит данные, необходимые Cluster API Provider Static (CAPS) для подключения по протоколу SSH. CAPS подключается к серверу (виртуальной машине), описанному в ресурсе [StaticInstance](cr.html#staticinstance) для управления его состоянием.
+            Ссылка на этот ресурс указывается в параметре [credentialsRef](cr.html#staticinstance-v1alpha1-spec-credentialsref) ресурса `StaticInstance`.
```
Comment (kirkonru) [inline modules/040-node-manager/crds/sshcredentials.yaml:25]:
```suggestion
            Contains the data required by Cluster API Provider Static (CAPS) to connect over SSH. CAPS connects to the server (virtual machine) defined in the [StaticInstance](cr.html#staticinstance) custom resource to manage its state.
            A reference to this resource is specified in the [credentialsRef](cr.html#staticinstance-v1alpha1-spec-credentialsref) parameter of the `StaticInstance` resource.
```

**Review by kirkonru — APPROVED** [review]

File: modules/040-node-manager/crds/sshcredentials.yaml [hunk 6]
```diff
@@ -21,9 +21,8 @@ spec:
       schema:
         openAPIV3Schema:
           description: |
-            SSHCredentials is the Schema for the SSH credentials to connect to the hosts via SSH.
-
-            The `StaticInstance` resource references this resource.
+            Contains the data required by Cluster API Provider Static (CAPS) to connect over SSH. CAPS connects to the server (virtual machine) defined in the [StaticInstance](cr.html#staticinstance) custom resource to manage its state.
+            A reference to this resource is specified in the [credentialsRef](cr.html#staticinstance-v1alpha1-spec-credentialsref) parameter of the `StaticInstance` resource.
```
Comment (z9r5) [inline modules/040-node-manager/crds/sshcredentials.yaml:25]:
```suggestion
            Contains the data required by Cluster API Provider Static (CAPS) to connect over SSH. CAPS connects to the server (virtual machine) defined in the [StaticInstance](cr.html#staticinstance) custom resource to manage its state.
            
            A reference to this resource is specified in the [credentialsRef](cr.html#staticinstance-v1alpha1-spec-credentialsref) parameter of the `StaticInstance` resource.
```

File: modules/040-node-manager/crds/sshcredentials.yaml [hunk 7]
```diff
@@ -21,9 +21,9 @@ spec:
       schema:
         openAPIV3Schema:
           description: |
-            SSHCredentials is the Schema for the SSH credentials to connect to the hosts via SSH.
-
-            The `StaticInstance` resource references this resource.
+            Contains the data required by Cluster API Provider Static (CAPS) to connect over SSH. CAPS connects to the server (virtual machine) defined in the [StaticInstance](cr.html#staticinstance) custom resource to manage its state.
```
Comment (z9r5) [inline modules/040-node-manager/crds/sshcredentials.yaml:24]:
```suggestion
            Contains credentials required by Cluster API Provider Static (CAPS) to connect over SSH. CAPS connects to the server (virtual machine) defined in the [StaticInstance](cr.html#staticinstance) custom resource to manage its state.
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6845](https://github.com/deckhouse/deckhouse/pull/6845) to the branch [release-1.55](https://github.com/deckhouse/deckhouse/tree/release-1.55) successful!

**Pull request merged**